Monika After Story/game/script-stories.rpy

```diff
@@ -10,11 +10,16 @@
 # dict of tples containing the stories event data
 default persistent._mas_story_database = dict()
 default mas_can_unlock_story = False
+default mas_can_unlock_scary_story = False
+default mas_full_scares = False

```

Comment:
is this supposed to be the whether or not like they horrors/spooks?

if so, then mark it with a todo/comment so we can change this when the o31 stuff gets in.

Reply:
Nope it's just for easy testing